kvserver/rangefeed: rename Send to SendUnbuffered

wenyihu6 opened the pull request:
**kvserver/rangefeed: move helpers to processor_helpers_test**

This patch moves the helper functions in processor_test.go to
processor_helpers_test.go.

Part of: https://github.com/cockroachdb/cockroach/issues/126560
Release note: none

---

**kvserver/rangefeed: rename Send to SendUnbuffered for Stream**

This patch renames Send to SendUnbuffered in Stream to distinguish
it from SendBuffered future patches are going to introduce.

Part of: https://github.com/cockroachdb/cockroach/issues/129813
Release note: none

---

**kvserver/rangefeed: rename SendIsThreadSafe to SendUnbufferedIsThreadSafe for Stream**

This patch renames SendIsThreadSafe to SendUnbufferedIsThreadSafe
in Stream.

Part of: https://github.com/cockroachdb/cockroach/issues/129813
Release note: none

wenyihu6 committed:
kvserver/rangefeed: move helpers to processor_helpers_test

This patch moves the helper functions in processor_test.go to
processor_helpers_test.go.

Epic: none
Release note: none

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/129229)
<!-- Reviewable:end -->

wenyihu6 committed:
kvserver/rangefeed: rename Send to SendUnbuffered for Stream

This patch renames Send to SendUnbuffered in Stream to distinguish
it from SendBuffered future patches are going to introduce.

Part of: #129813
Release note: none

wenyihu6 committed:
kvserver/rangefeed: rename SendIsThreadSafe to SendUnbufferedIsThreadSafe for Stream

This patch renames SendIsThreadSafe to SendUnbufferedIsThreadSafe
in Stream.

Part of: #129813
Release note: none

nvb reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> Reviewed 3 of 3 files at r1, 12 of 12 files at r2, 10 of 10 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/129229#-:-O5Z8_Pc9WThQdZGeEUj:bnfp4nl)* status: <span>:shipit:</span> complete! 1 of 0 LGTMs obtained (waiting on @wenyihu6)



<!-- Sent from Reviewable.io -->

wenyihu6 commented:
TFTR!

bors r=nvanbenschoten
